lvgl example app doesn't build

```
$ make
  CC        lvgl_driver.c
In file included from lvgl_driver.c:1:
lvgl_driver.h:5:10: fatal error: lvgl/lvgl.h: No such file or directory
 #include <lvgl/lvgl.h>
          ^~~~~~~~~~~~~
compilation terminated.
make: *** [build/cortex-m0/lvgl_driver.o] Error 1
```

We should update this so that examples "just work", or remove it as an example.

--- a/examples/lvgl/Makefile
+++ b/examples/lvgl/Makefile
@@ -18,3 +18,10 @@ APP_HEAP_SIZE := 20000
 # Include userland master makefile. Contains rules and flags for actually
 # building the application.
 include $(TOCK_USERLAND_BASE_DIR)/AppMakefile.mk
+
+# Ensure the required submodule is present
+lvgl_driver.c:	| ../../lvgl/lvgl/lvgl.h
+
+../../lvgl/lvgl/lvgl.h:
+	git submodule init ../../lvgl/lvgl
+	git submodule update ../../lvgl/lvgl

